okhttp/src/main/java/okhttp3/internal/http/CallServerInterceptor.java

```diff
@@ -99,6 +99,25 @@ public CallServerInterceptor(boolean forWebSocket) {
         .responseHeadersEnd(realChain.call(), response);
```

Comment:
I'm pondering the consequences of calling this 2x. It's going to crash listeners.

Can you change your fix to go before this listener call back? And we won't call back on the headers for the unsolicited 100; it's not worth the trouble.

Reply:
Okay, I changed this accordingly in 785e1d4